AndelaOSP/andela-societies-frontend
# Resolves #24 - Add ability to sign in

(Feature #24) User should be able to sign in via the andela api

**Description**
A user should be able to sign in into the application
- When the user clicks on the sign in button,  the user is redirected to a Google Signin via the Andela API. Once the correct credentials are provided a token is set in a cookie. The token is checked for validity i.e, has not expired, has Andelan role
- If token also has Fellow role, the user is sent to the my-activities page, otherwise user is redirected to  the sign in page

**How to test**
* Add `soc-dev.andela.com` to `/etc/hosts`
- Run the application
- With a correct fellow email address, one should be able to sign in and access the my activities page
- With a non-Andelan gmail address one should see the sign in error notification

<img width="1009" alt="screen shot 2018-03-15 at 10 31 08" src="https://user-images.githubusercontent.com/13675851/37449653-16cde5b0-283c-11e8-8fb2-37bf587683a7.png">
